app/templates/search/_opportunity_data.html

```diff
@@ -0,0 +1,18 @@
+<br />
+<div class="dmspeak">
+  <h3 class="heading-small opportunity-data-header">Opportunity data</h3>
```

Comment:
Is this `opportunity-data-header` class here purely for the benefit of the tests?  If so then I'd say drop it and find a different way to identify the element in the test.

Reply:
Ahh, that opens a good question: what is a good reason to make a class? Is giving a specific name to an element a good reason? Or does it need to introduce some new .scss?

Reply:
I don't have strong opinions on this particularly, again some deep front-end philosophy is involved...   I'm pretty sure that if it's giving a name to an element then it should be an ID rather than a class.  Classes should be applicable to possibly multiple elements on a single page.  I also think there's no point in creating a class unless it has some use - so if there's no associated style then it's a waste of everyone's bandwidth to download the unnecessary markup.

Reply:
In principle I agree with you that id should be used for unique element and this is how I have been doing stuff in my personal projects. Yet since I started working at DM I noticed we don't really use id's, just classes. 

As to giving elements unique names, I find it helps me navigate around the template better. 

Also, this probably is not a good argument, but we do have empty classes already. For example now I wanted to add padding to "search-page-filters" class, but it turned out such class does not exist in frontend-toolkit 😬 .
[edit: I found the class in buyer-frontend in "digitalmarketplace-buyer-frontend/app/assets/scss/_search-page.scss". Is it ok that there is a whole lot of things that is not in frontend-toolkit? 😕 

Very open to changing class names to id names, or alternatively to changing class names to different class names but not sure what those alternative class names could be.